src/workers/parser/utils/ast.js

```diff
@@ -14,28 +14,28 @@ import { getSource } from "../sources";
 let ASTs = new Map();
 
 function _parse(code, opts) {
-  return babylon.parse(code, opts);
+  return babylon.parse(code, {
+    ...opts,
+    tokens: true
+  });
 }
 
 const sourceOptions = {
-  generated: {
-    tokens: true
-  },
+  generated: {},
   original: {
     sourceType: "unambiguous",
-    tokens: true,
     plugins: [
       "jsx",
       "flow",
       "doExpressions",
       "objectRestSpread",
       "classProperties",
-      "exportExtensions",
+      "exportDefaultFrom",
+      "exportNamespaceFrom",
```

Comment:
This plugin got split in two in 7.x.

Reply:
There are new plugins in 7.x that I think we could enable like the TypeScript one

Reply:
Can't enable both `flow` and `typescript` at the same time, so there'd need to be some logic to check the original file extensions, but should otherwise be workable.